tests/bench_msg_pingpong/Makefile.ci

```diff
@@ -0,0 +1,2 @@
+BOARD_INSUFFICIENT_MEMORY := \
+    nucleo-f031k6
```

Comment:
maybe change this to:

```
    nucleo-f031k6 \
    #
```

to match structure of other files and make additions easy